readme.md

```diff
@@ -25,8 +25,8 @@ ___
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://AndriiKasiian.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://AndriiKasiian.github.io/layout_search-bar-airbnb/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="951" alt="image" src="https://github.com/user-attachments/assets/6845ebf6-4491-4ad8-8c2a-a57fd90bab52">